python/tests/api/pyspark/experimental/test_profiler_function.py

```diff
@@ -3,6 +3,8 @@
 from typing import Dict
 
 import pytest
+
+# from pyspark import SparkConf
```

Comment:
Remove since it's commented out?

Reply:
One hypothesis about the cause of the strange Spark test behavior was that we needed to tweak the Spark configuration to use more memory. This file will need a lot of cleanup once we figure out what to do about the Spark weirdness.